_events/1.markdown

```diff
@@ -1,12 +1,26 @@
 ---
 layout: event
 title:  "2019-07-02 Call"
-date:   2019-07-02 9:00
-file_list:
-  # google drive links
-  test.png: Praesent sed mi at orci pretium hendrerit. Vestibulum pharetra laoreet mi id maximus.
-  https://www.foo.com: Aenean accumsan consequat augue sit amet porttitor.
-summary: This is a summary of the call.
+date:   2019-07-02 10:00
+file_list:HCA metadata schema: https://github.com/HumanCellAtlas/metadata-schema
```

Comment:
```
file_list:
  HCA metadata schema: https://github.com/HumanCellAtlas/metadata-schema
  HCA schema validator: https://github.com/HumanCellAtlas/ingest-validator-js
 .
 .
 .
```

I think it'd be more readable (and functional) to make this look like https://raw.githubusercontent.com/hms-dbmi/x-consortium/master/_example_templates/0101.markdown i.e an indented list under `file_list`